core/domain/suggestion_registry.py

```diff
@@ -64,11 +67,12 @@ def to_dict(self):
             'target_id': self.target_id,
             'target_version_at_submission': self.target_version_at_submission,
             'status': self.status,
-            'author_id': self.author_id,
+            'author_name': self.get_author_name(),
```

Comment:
make sure that we consistent with this everywhere.

Reply:
I edited all the tests that checked the dict and made sure they pass. I think we are consistent.